overlord/snapstate/check_snap_test.go

```diff
@@ -1192,6 +1209,76 @@ system-usernames:
 	}
 }
 
+func (s *checkSnapSuite) TestCheckSnapSystemUsernamesCallsSnapDocker(c *C) {
```

Comment:
The test looks identical to the one above (TestCheckSnapSystemUsernamesCallsSnapDaemon) except the slightly  yaml - maybe we can use the "Test1, Test2, testCommon" pattern here, something like: https://paste.ubuntu.com/p/8FCxhhkkzH/

Reply:
sure, thanks for the patch

Reply:
applied